pyaedt/edb_core/configuration.py

```diff
@@ -0,0 +1,201 @@
+import json
+
+from pyaedt.edb_core.edb_data.simulation_configuration import SimulationConfiguration
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+class Configuration:
+    """Enables export and import of a JSON configuration file that can be applied to a new or existing design."""
+
+    def __init__(self, pedb):
+        self._pedb = pedb
+
+    @pyaedt_function_handler
+    def load(self, config_file):
+        """Import configuration settings from a JSON file and apply it to the current design.
+
+        Parameters
+        ----------
+        config_file : str, SimulationConfiguration
+            Full path to json file.
+
+        Returns
+        -------
+        dict
+            Config dictionary.
+
+        Examples
+        --------
+        >>> from pyaedt import Edb
+        >>> edbapp = Edb("ansys_pcb.aedb")
+        >>> edbapp.configuration.load("configure.json")
+        """
+
+        if isinstance(config_file, SimulationConfiguration):
```

Comment:
argument for load method is called config_file but can also accept SimulationConfiguration object. I would rename the argument. Can we rather embed the new json file into SimulationConfig class (inherrited) so that we handle only one json for all ?

Reply:
@svandenb-dev as we discussed, I will remove SimulationConfig support from this method.